app/scripts/modules/core/src/utils/timeFormatters.ts

```diff
@@ -30,7 +30,10 @@ export function timestamp(input: any) {
     return '-';
   }
   const tz = SETTINGS.defaultTimeZone;
-  const thisMoment = moment.tz(parseInt(input, 10), tz);
+  const thisMoment = moment
+    .tz(parseInt(input, 10), tz)
+    .clone()
+    .tz(moment.tz.guess());
```

Comment:
Why `clone` and then call `tz` again? What happens if you just call `moment.tz(parseInt(input, 10), moment.tz.guess())`?

Reply:
Fixed in the next commit 582bc9e